excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/parser/AbstractDisplayGroupParser.java

```diff
@@ -141,9 +139,28 @@ public AbstractDisplayGroupParser(ParseContext parseContext, ShowConditionParser
 
         group.addDisplayGroupCaseFields(groupCaseFields);
         this.groupShowConditionColumn.ifPresent(column -> parseGroupShowCondition(column, group, groupDefinition.getValue()));
+        this.userRoleColumn.ifPresent(column -> parseUserRole(caseType, group, groupDefinition.getValue()));
         return ParseResult.Entry.createNew(group);
     }
 
+    private void parseUserRole(CaseTypeEntity caseType,
+                               final DisplayGroupEntity group,
+                               final  List<DefinitionDataItem> groupDefinition) {
+        if (groupDefinition.stream().filter(ddi -> ddi.getString(USER_ROLE) != null).count() > 1) {
```

Comment:
`ddi.getString(USER_ROLE) != null`
not sure but do we have to check for empty string as well?

Reply:
It will just change the behaviour, I changed it anyway. It used to return an error for cells with spaces, now we ignore those cells.

Reply:
It will just change the behaviour, I changed it anyway. It used to return an error for cells with spaces, now we ignore those cells.